api/src/services/tm_types/map_roulette.js

```diff
@@ -0,0 +1,133 @@
+const rp = require('request-promise-native')
+const limit = require('p-limit')(5)
+// const extractCampaignHashtag = require('../../utils/extractCampaignHashtag')
+
+/**
+ * Methods to grab data from MapRoulette API
+ */
+
+class MapRouletteAPI {
+  /**
+   * Creates an instance of MapRouletteAPI.
+   * @param {*} { url, taskerId, apiUrl, opts }
+   * @memberof MapRouletteAPI
+   */
+  constructor ({ url, taskerId, apiUrl, opts }) {
+    this.url = url
+    this.api_url = apiUrl
+    this.tasker_id = taskerId
+    this.opts = opts || {}
+  }
+
+  getUrlForProject (id) {
+    return `${this.url}/project/${id}`
+  }
+
+  getLastUpdated (id) {
+    return this.getProject(id).then(data => {
+      return JSON.parse(data)
+    }).then(parsed => {
+      return parsed.lastUpdated
+    })
+  }
+
+  /*
+   * For consistency, use Projects to refer to Scoreboard projects
+   * MapRoulette uses Projects :: Challenges
+   * Challenges are treated as Campaigns.
+   * TODO this currently statically returns the first 50 objs
+  */
+  async getProjects () {
```

Comment:
It looks like there are a few campaigns/challenges coming through without any features. 
I only noticed as I was paginating through the `mr` campaigns in the UI. They show up around the 4th page.

Reply:
hm strange. looks like on first load of the page, no geometry shows up in the map but if you click on a campaign, the geometry appears. but when I look at the response, all the entries have geometry.

Reply:
On the client side, the geometry is getting delivered on first load. so looks like there is some bug preventing it from display on the initial view.

Reply:
Maybe this is better addressed in a different PR?